<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createFiscalYear</class>
 <widget class="QDialog" name="createFiscalYear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Accounting Period(s)</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QStackedWidget" name="_stack">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="_noPrivPage">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="XLabel" name="_noPrivLit">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You do not have permission to create Fiscal Years and Accounting Periods. Please ask an administrator to create them or grant you the &lt;span style=&quot; font-style:italic;&quot;&gt;MaintainAccountingPeriods&lt;/span&gt; privilege.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_newPeriodPage">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="DateCluster" name="_period"/>
       </item>
       <item row="0" column="0">
        <widget class="XLabel" name="_noFYLit">
         <property name="text">
          <string>There is no Fiscal Year to put a new period in. Please choose one of the other options above.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_newFYPage">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="XLabel" name="_firstDayLit">
         <property name="text">
          <string>First Day of the Fiscal Year:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="DLineEdit" name="_firstDay"/>
       </item>
       <item row="1" column="0">
        <widget class="XLabel" name="_styleLit">
         <property name="text">
          <string>Type of Fiscal Year:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="XComboBox" name="_style"/>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>26</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="_copyFYPage">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0" colspan="2">
        <widget class="XTreeWidget" name="_fy"/>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="XLabel" name="_noFYToCopyLit">
         <property name="text">
          <string>There is no Fiscal Year to copy. Please choose one of the other options above.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="_save">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your system does not have a valid or open Accounting Period for the current date. You should define account periods before posting any transactions in the system.&lt;/p&gt;&lt;p&gt;Please select one of the options below, fill in the necessary data, and click &lt;span style=&quot; font-weight:600;&quot;&gt;Save&lt;/span&gt;. If you &lt;span style=&quot; font-weight:600;&quot;&gt;Cancel&lt;/span&gt;, you will need to create accounting periods with &lt;span style=&quot; font-weight:600;&quot;&gt;Accounting&lt;/span&gt; &amp;gt; &lt;span style=&quot; font-weight:600;&quot;&gt;Fiscal Calendar&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="XComboBox" name="_choose"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateCluster</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
  </customwidget>
  <customwidget>
   <class>DLineEdit</class>
   <extends>QWidget</extends>
   <header>datecluster.h</header>
  </customwidget>
  <customwidget>
   <class>XComboBox</class>
   <extends>QComboBox</extends>
   <header>xcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>XLabel</class>
   <extends>QLabel</extends>
   <header>xlabel.h</header>
  </customwidget>
  <customwidget>
   <class>XTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>xtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
